docs/guides/06-errors.md

```diff
@@ -264,6 +264,20 @@ paths:
 
 **Explanation:** This error occurs when the current request specifies a `Content-Type` that isn't supported by corresponding HTTP Operation. In the case there is no request body or `Content-Length` is 0, the `Content-Type` header is ignored.
 
+
+### SCHEMA_TOO_COMPLEX
+
+**Message: Unable to generate [body|header] for response. The schema is too complex to generate.**
+
+**Returned Status Code: `500`**
+
+**Explanation:** This error occurs when part of the response can't be generated using JSON Schema Sampler. JSON Schema Sampler has been configured to use a limit of 2500 "ticks". A "tick" is loosely defined as any instance of a JSON Schema schema or subschema. This includes a/an single:
```

Comment:
Recommend: "This error occurs when part of the response can't be generated using **the** JSON Schema Sampler. **The** JSON Schema Sample **has** a limit of 2500 ticks."

"Has been configured" makes it feel open to changing.

Reply:
it is open to changing